src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -13,6 +18,17 @@ const StyledField = styled(Field)`
   }
 `;
 
+const StyledEndIconContainer = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 99%;
+  transform: translate(-99%, -50%);
+`;
+
+const StyledMediaInput = styled(MediaInput)`
+  padding-right: ${(props) => props.theme.space.md};
```

Comment:
It feels to me that the input is a bit too close to the icon. Why are we setting it specifically to `space.md`? The container has a padding of 10px, we are adding 20px more reaching 30px, but the icon button is 32px

Reply:
I believe I saw the incorrect values on the design. I will fix it 👍